src/ui/Base/TopNavigation.tsx

```diff
@@ -1,40 +1,53 @@
-import { IconArrowLeft, IconHamburger } from '@public/icons';
-import Link from 'next/link';
-import { PropsWithChildren } from 'react';
-
-interface Props extends PropsWithChildren {
-  titleAlign?: 'center' | 'left';
-  title?: string;
-  isOwner?: boolean;
+import { PropsWithChildren, ReactNode } from 'react';
+
+interface TopNavigationProps {
+  children?: ReactNode;
 }
 
-const TopNavigation = ({
-  children,
-  titleAlign = 'center',
-  title = '',
-  isOwner = false,
-}: Props) => {
+type ItemProps = TopNavigationProps;
+
+const TopNavigation = ({ children }: TopNavigationProps) => {
   return (
-    <div className="relative flex h-[5.4rem] w-full gap-[1.5rem] bg-opacity-0 px-[2rem] py-[1.7rem]">
-      <Link href=".">
-        <IconArrowLeft className="hover:cursor-pointer" />
-      </Link>
-      <div
-        className={`flex w-full pr-[3.5rem] text-md font-normal leading-[1.9rem] ${TITLE_ALIGN_CLASSES[titleAlign]}`}
-      >
-        {title}
-      </div>
-      <div className="absolute right-[2rem] flex gap-[1rem]">
-        {children}
-        {isOwner && <IconHamburger className="hover:cursor-pointer" />}
-      </div>
+    <div className="relative flex h-[2.4rem] w-full items-center justify-center bg-opacity-0 px-[2rem] py-[1.7rem] text-md">
+      {children}
     </div>
   );
 };
 
-export default TopNavigation;
+const LeftItem = ({ children }: ItemProps) => {
+  return (
+    <div className="absolute left-[2rem] [&_svg]:h-[2rem] [&_svg]:w-[2rem] [&_svg]:cursor-pointer">
+      {children}
+    </div>
+  );
+};
+
+type CenterItemProps = PropsWithChildren<{ textAlign?: 'left' | 'center' }>;
+
+const textAligns = {
+  left: 'text-left',
+  center: 'text-center',
+} as const;
 
-const TITLE_ALIGN_CLASSES = {
-  center: 'justify-center',
-  left: 'justify-start',
+const CenterItem = ({ children, textAlign = 'center' }: CenterItemProps) => {
+  const alignClassName = textAligns[textAlign];
+  return (
+    <div className={`w-full px-[3.5rem] ${alignClassName}`}>{children}</div>
```

Comment:
text-overflow 될 경우를 대비해서 `truncate` 클래스 추가하면 좋을 것 같아요! [(참고)](https://tailwindcss.com/docs/text-overflow#truncate)

```suggestion
    <div className={`w-full px-[3.5rem] truncate ${alignClassName}`}>{children}</div>
```

Reply:
오 좋네요, 그런데 텍스트를 일부분만 보여주는게 UX 관점에서 좋은지는 모르겠어요.

가령 상세페이지 같은 곳에서 해당 컴포넌트가 쓰일 수 있을 것 같은데, 상세 페이지인데 내용의 일부분이 잘리면 사용자는 어디서 전체 내용을 봐야할까요? 🤔

Reply:
@gxxrxn 오 좋네요, 그런데 텍스트를 일부분만 보여주는게 UX 관점에서 좋은지는 모르겠어요.

가령 상세페이지 같은 곳에서 해당 컴포넌트가 쓰일 수 있을 것 같은데, 상세 페이지인데 내용의 일부분이 잘리면 사용자는 어디서 전체 내용을 봐야할까요? 🤔

Reply:
@minjongbaek 메인 컨텐츠를 보여주는 영역에서 텍스트 전체를 보여주면 좋지 않을까요?! 상단 네비게이션에서 2줄 이상의 텍스트를 보여주기엔 한계가 있을 것 같고, 잘리는게 싫다면 텍스트가 rolling 되도록 구현해볼 수 있을 것 같아요!

Reply:
@gxxrxn 다른 서비스를 보니 콘텐츠를 보여주는 페이지에서는 GNB에 아무런 텍스트를 넣어주지 않는 방법도 있을 것 같아요 ![image](https://github.com/prgrms-web-devcourse/Team-Gaerval-Dadok-FE/assets/48359052/0412ad62-4243-4dcf-b059-b9bc1f0a040a)

Reply:
@minjongbaek @hanyugeon 
고민해보니 제가 참고했던 서비스는 타이틀이 상대적으로 중요했기 때문에 GNB 영역에서 보여줬던 것 같아요!(밴드 모임 쪽 UI를 참고했습니다!) 민종님 말씀처럼 텍스트 없이 GNB를 구성하는 것도 좋아 보입니다!! 재현님 생각은 어떠신가요?!

Reply:
피그마 기준으로 글 작성하기 처럼 새로운 레이어가 생기는 경우에는 보여주면 좋을 것 같아요! 아래 사진처럼요!

![image](https://github.com/prgrms-web-devcourse/Team-Gaerval-Dadok-FE/assets/48359052/d6881c09-a3df-4516-be43-236be86f5d87)

Reply:
책 이름, 모임 이름 등의 타이틀을 보여주는 상세 페이지에서는 제거하고, 글 작성 / 댓글 수정을 위한 레이아웃에서는 보여주자는 말씀이시죠?

Reply:
@gxxrxn 네 맞아요! 일관성이 없어 보일 수는 있지만, 폼, 키보드, 액션 버튼만 있는 UI에서는 텍스트를 보여주는게 좋을 것 같다고 생각해요.

Reply:
좋습니다~

Reply:
comment;

@minjongbaek @gxxrxn 
`그런데 텍스트를 일부분만 보여주는게 UX 관점에서 좋은지는 모르겠어요.` 이 부분에 동감합니다!
피그마 디자인을 확인해보니 책 이름, 모임 이름, 유저 프로필, 등... GNB에 타이틀은 빠져도 좋을 것 같네요!!

Reply:
@hanyugeon @gxxrxn 그럼 이대로 머지하겠습니다~

Reply:
@minjongbaek 네넵